### PR TITLE
vladimir s.

### PR DESCRIPTION
**Pull request by svladimir1010**
something like this

### CODE REVIEW

**Comment by svladimir1010** [comment]
I think so everything

File: students.txt [hunk 1]
```diff
@@ -22,6 +22,7 @@ Hobbies:
  - wine making and degustating
  - literature
  - teaching children
+ - spend time with my family or friends
```
Comment (vit-lebediev) [inline students.txt]: You've added an item to someone else's description. This was NOT the task.

You need to add a list of your hobbies, plus extend your last name on line 13.

**Review by vit-lebediev — CHANGES_REQUESTED** [review]
This is not what was required in the task. Please review my comments and update PR accordingly.

**Comment by svladimir1010** [comment]
Здесь что то не так?

**Pull request closed without merging**